packages/driver/src/util/serialization.ts

```diff
@@ -0,0 +1,38 @@
+import _ from 'lodash'
+import structuredClonePonyfill from 'core-js-pure/actual/structured-clone'
+
+// If a native structuredClone exists, use that to determine if a value can be serialized or not. Otherwise, use the ponyfill.
+// we need this because some implementations of SCA treat certain values as unserializable (ex: Error is serializable in ponyfill but NOT in firefox implementations)
+// @ts-ignore
+const structuredCloneRef = window?.structuredClone || structuredClonePonyfill
```

Comment:
move the `stucturedClone()` serialization items here as I will need them here for the error refactor where this code is shared